app/controllers/internal/configs_controller.rb

```diff
@@ -79,11 +80,13 @@ def extra_authorization_and_confirmation
 
   def clean_up_params
     config = params[:site_config]
-    config[:suggested_tags] = config[:suggested_tags].downcase.delete(" ") if config[:suggested_tags]
     config[:sidebar_tags] = config[:sidebar_tags].downcase.delete(" ") if config[:sidebar_tags]
+    config[:suggested_tags] = config[:suggested_tags].downcase.delete(" ") if config[:suggested_tags]
+    config[:suggested_users] = config[:suggested_users].downcase.delete(" ") if config[:suggested_users]
```

Comment:
Maybe at one point we can change this to make it less repetitive?

```ruby
%i[sidebar_tags suggested_tags suggested_users].each do |param|
  config[param] = config[param].downcase.delete(" ") if config[param]
end
```

Reply:
Ah of course. I was trying to not do too much refactor for this PR, but this seems obviously better, so I've fixed this 👍